css/styles.scss

```diff
@@ -263,3 +263,63 @@ body {
   background-image: linear-gradient(360deg, transparent, $san-juan);
   position: absolute;
 }
+
+/*Model X*/
+
+/*Third Sector*/
+.model-x-top-sector {
+  @include dim(100%, 100%);
+  .model-x-top-sector-img {
+    width: 100%;
+    height: 70% !important;
+    background-position: center;
+    object-fit: cover !important;
+  }
+  .top-sector-head {
+    h6 {
+      font-weight: 400;
```

Comment:
I see you're repeating the same value for the font weight... you may save it to a variable